xml/Microsoft.CLRAdmin/IDataObject.xml

```diff
@@ -129,6 +153,7 @@
         <summary>This API supports the .NET Framework infrastructure and is not intended to be used directly from your code.</summary>
         <returns>To be added.</returns>
         <remarks>To be added.</remarks>
+        <forInternalUseOnly />
```

Comment:
Is this API going to be covered in a different PR or here? If in a different one, then this change should be done there.